tabbycat/notifications/utils.py

```diff
@@ -30,23 +37,23 @@
 }
 
 
-def _assemble_panel(adjs):
+def _assemble_panel(adjs: List[Tuple['Person', str]]) -> str:
     adj_string = []
     for adj, pos in adjs:
         adj_string.append("%s (%s)" % (adj.name, adj_position_names[pos]))
 
     return ", ".join(adj_string)
 
 
-def _check_in_to(pk, to_ids):
+def _check_in_to(pk, to_ids) -> bool:
     try:
         to_ids.remove(pk)
     except KeyError:
         return False
     return True
 
 
-def adjudicator_assignment_email_generator(to, url, round):
+def adjudicator_assignment_email_generator(to: 'QuerySet[Person]', url: str, round: 'Round') -> List[Tuple[Dict[str, str], 'Person']]:
```

Comment:
This is ugly... The generators could use dataclasses instead, which would then show what properties (and types) would be present for each.

As an example, the dataclass for this would be like:
```py
from dataclasses import dataclass

@dataclass
class AdjudicatorAssignment:
    ROUND: str
    VENUE: str
    PANEL: str
    DRAW: str
    USER: str
    POSITION: str
    URL: str
```
etc. It would be more consequential for the more complicated generators though.